libtmux/window.py

```diff
@@ -434,7 +434,10 @@ def split_window(
             'window_index',
             'window_id',
         ] + formats.PANE_FORMATS
-        tmux_formats = ['#{%s}\t' % f for f in pformats]
+        tmux_formats = [
+            ('#{%%s}%s' % formats.FORMAT_SEPERATOR) % f
```

Comment:
This is interesting, any ideas for cleaning it up?

Perhaps a [PEP 498](https://www.python.org/dev/peps/pep-0498/) "f-string"?